initramfs-init.in

```diff
@@ -646,6 +657,23 @@ if [ -n "$ovlfiles" ]; then
 else
 	apk add --root $sysroot $repo_opt $apkflags $pkgs
 fi
+
+if [ -n "$KOPT_ssh_key" ]; then
+	rc_add sshd default
+	echo "PermitRootLogin prohibit-password" >> $sysroot/etc/ssh/sshd_config
+	mkdir -p $sysroot/root/.ssh
+	case "$KOPT_ssh_key" in
+		http://*|https://*) wget "$KOPT_ssh_key" -O $sysroot/root/.ssh/authorized_keys;;
```

Comment:
As reported on [bugs](https://bugs.alpinelinux.org/issues/8917) we should depend on GNU wget instead of using busybox wget. GNU wget should be the default for whenever we want to fetch from a secure server (ie modloop and others).
And what happens here if i use ftp(s)?

Reply:
Updated everything in new commit. Thanks.